src/cogs/help.py

```diff
@@ -40,24 +40,34 @@ def get_bot_help(
         mapping: Mapping[Optional[commands.Cog], List[commands.Command]]
     ) -> discord.Embed:
         """Generates the embed for the bot help command."""
-        out = discord.Embed(title="Help")
+        out = discord.Embed(
+            title="Help",
+            description="To find out more info, use **!help [Cog/Command]**. Note that this is case sensitive.",
+        )
 
         # this might be able to be cleaned up.
         freestanding_commands = []
         for cog, cmds in mapping.items():
             if cog is None:
                 freestanding_commands.extend(cmds)
                 continue
-            out.add_field(
-                name=cog.qualified_name,
-                value=f"{count(cog.walk_commands())} commands.",
-                inline=False,
-            )
+            command_count = count(cog.walk_commands())
+            if command_count != 0:
+                command_logic = (
+                    "command" if count(cog.walk_commands()) == 1 else "commands"
+                )
+                out.add_field(
+                    name=cog.qualified_name,
+                    value=f"```{count(cog.walk_commands())} {command_logic}.```",
+                    inline=True,
+                )
```

Comment:
You store the command_count (which is good), but you don't actually use it here. Refactor command_logic and the add_field to use the command_count variable.